CHANGELOG.md

```diff
@@ -36,6 +37,55 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.12.0]
+
+> Release date: 2022/04/21
+
+### Added
+
+- Konnect can now be configured via the main `deck` command, while
+  `deck konnect` is now considered deprecated.
+  [#645](https://github.com/Kong/deck/pull/645)
+- Added `--skip-ca-certificates` flag. When present, decK will not attempt to
+  sync CA certificates. This assists with using decK to manage multiple
+  workspaces. CA certificates do not belong to a specific workspace, but can be
+  seen and managed through workspaced endpoints, and decK will attempt to
+  remove them if they are not present in a state file when syncing a workspace,
+  even if they were created for use with configuration in another workspace.
+  [#617](https://github.com/Kong/deck/pull/617)
+- Users can no longer attempt to set default values for fields where a default value
+  does not make sense (such as fields with unique constraints, like `id`), and
+  will print an error indicating the restricted field.
+  [#613](https://github.com/Kong/deck/pull/613)
+- Universal binaries are now available for OS X.
+  [#585](https://github.com/Kong/deck/pull/585)
```

Comment:
Not an edit but a comment - should we update the decK installation doc to include this info? (that would be here: https://github.com/Kong/docs.konghq.com/blob/deck-1.12/src/deck/installation.md).